presto-main/src/main/java/com/facebook/presto/util/ContinuousWorkUtils.java

```diff
@@ -0,0 +1,274 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.util;
```

Comment:
Why not `com.facebook.presto.operator`,  then you could make below methods to be `package` visible. That would make methods like  `ContinuesWork#map` be prefered than in this util class.

Reply:
I think it's good to be able to use methods either from `ContinuesWork` or `ContinuesWorkUtils`. This is because not all composition methods are part of ` ContinuesWork` (e.g: `mergeSort`). In the `mergeSort` case you probably want to compose the classical way, e.g:
```
buildPage(mergeSorted(transform(...
```